requirements.txt

```diff
@@ -1,35 +1,33 @@
-aiohttp==3.9.5
+aiohappyeyeballs==2.4.4
+aiohttp==3.10.11
 aiosignal==1.3.1
-annotated-types==0.7.0
-async-timeout==4.0.3
-attrs==23.2.0
+async-timeout==5.0.1
+attrs==24.2.0
 autocommand==2.2.2
 backports.tarfile==1.2.0
-certifi==2024.6.2
-chardet==5.2.0
-charset-normalizer==3.3.2
+certifi==2024.8.30
 click==8.1.7
 discord.py==2.4.0
-frozenlist==1.4.1
-idna==3.7
-inflect==7.3.0
-irc==20.4.1
-jaraco.classes==3.4.0
-jaraco.collections==5.0.1
-jaraco.context==5.3.0
-jaraco.functools==4.0.1
+frozenlist==1.5.0
+idna==3.10
+importlib_resources==6.4.5
+irc==20.5.0
+jaraco.collections==5.1.0
+jaraco.context==6.0.1
+jaraco.functools==4.1.0
 jaraco.logging==3.3.0
-jaraco.stream==3.0.3
-jaraco.text==3.12.1
-more-itertools==10.3.0
-multidict==6.0.5
+jaraco.stream==3.0.4
+jaraco.text==4.0.0
+more-itertools==10.5.0
+multidict==6.1.0
 openttd-helpers==1.4.0
-pydantic==2.7.4
-pydantic_core==2.18.4
-pytz==2024.1
-sentry-sdk==2.7.1
-tempora==5.6.0
-typeguard==4.3.0
+propcache==0.2.0
+python-dateutil==2.9.0.post0
+pytz==2024.2
+sentry-sdk==2.19.0
+six==1.16.0
+tempora==5.7.0
 typing_extensions==4.12.2
-urllib3==2.2.2
-yarl==1.9.4
+urllib3==2.2.3
+yarl==1.15.2
+zipp==3.20.2
```

Comment:
Why is this dependency introduced? When I run an update, I don't see this one.

Reply:
Another polyfill lib, but only needed with Python 3.8. Not with 3.10, so now removed.